src/main/java/graphql/annotations/DefaultTypeFunction.java

```diff
@@ -41,11 +41,9 @@
 import static graphql.annotations.util.NamingKit.toGraphqlName;
 import static graphql.schema.GraphQLEnumType.newEnum;
 
-@Component(scope = ServiceScope.SINGLETON, property = "type=default")
+@Component(property = "type=default")
```

Comment:
Component will remain a singleton since `serviceFactory=false` from DS 1.2 will be used.